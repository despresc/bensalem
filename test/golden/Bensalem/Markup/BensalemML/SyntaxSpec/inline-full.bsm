@inline[attr1={thing}]{argument}
