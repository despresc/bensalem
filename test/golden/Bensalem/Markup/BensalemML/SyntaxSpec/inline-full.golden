fromList
 [ ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 0
     , srcLine = 1
     , srcCol = 1
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 7
     , srcLine = 1
     , srcCol = 8
     }
    }
   , elementName = "inline"
   , elementAttrs = Attrs
    ( fromList
     [
      ( Located
       { locatedSpan = SrcSpan
        { srcSpanName = ""
        , srcSpanStart = SrcPos
         { srcOffset = 8
         , srcLine = 1
         , srcCol = 9
         }
        , srcSpanEnd = SrcPos
         { srcOffset = 13
         , srcLine = 1
         , srcCol = 14
         }
        }
       , locatedVal = "attr1"
       }
      , BracedAttrVal
       ( fromList
        [ PlainText "thing" ]
       )
      )
     ]
    )
   , elementScopeContent = InlineScopeContent
   }
  )
 , GroupNode
  ( Group
   { groupStart = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 22
     , srcLine = 1
     , srcCol = 23
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 23
     , srcLine = 1
     , srcCol = 24
     }
    }
   , groupEnd = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 31
     , srcLine = 1
     , srcCol = 32
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 32
     , srcLine = 1
     , srcCol = 33
     }
    }
   , groupContent = fromList
    [ PlainText "argument" ]
   }
  )
 , LineEnd
 ]